src/main/resources/application.yml

```diff
@@ -47,6 +48,7 @@ spring:
    rest:
       # this points at the local WireMock server in the test environment.
       iexBasePath: http://localhost:${wiremock.server.port}/
+      iexToken: pk_8bf2fa8153b64102b1d34199273d2063/
```

Comment:
don't commit secrets to code repos, this should remain in a local property file that is not included in the git